HADOOP-19608. Upgrade to JUnit 5.13.3 + Surefire 3.5.3

steveloughran opened the pull request:
HADOOP-19608

* junit.jupiter and junit.vintage => 5.13.3
* junit.platform => 1.13.3
* Surefire => 3.5.3. Without that tests weren't being found.

### How was this patch tested?

Worked on the iceberg bulk delete branch for its new parameterized test class.
Not yet tested anything else ... let's see what yetus says. I'll do the ITest run

### For code changes:

- [X] Does the title or this PR starts with the corresponding JIRA issue id (e.g. 'HADOOP-17799. Your PR title ...')?
- [x] Object storage: have the integration tests been executed and the endpoint declared according to the connector-specific documentation?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the `LICENSE`, `LICENSE-binary`, `NOTICE-binary` files?

steveloughran committed:
HADOOP-19608. Upgrade to Junit 5.13.3

* junit.jupiter and junit.vintage => 5.13.3
* junit.platform => 1.13.3
* Surefire => 3.5.3. Without that tests weren't being found.
* cut a duplicate and conflicting import of mockito-jupiter;
  maven was warning of this.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 30s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  xmllint  |   0m  0s |  |  xmllint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  24m 28s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 16s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  compile  |   0m 16s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  mvnsite  |   0m 17s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   0m 21s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   0m 17s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  |  46m 31s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m  8s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m  9s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javac  |   0m  9s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 10s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  javac  |   0m 10s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 13s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 11s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   0m 11s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| -1 :x: |  shadedclient  |   1m 54s |  |  patch has errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 10s |  |  hadoop-project in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 21s |  |  The patch does not generate ASF License warnings.  |
|  |   |  51m  6s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.51 ServerAPI=1.51 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/7785 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient codespell detsecrets xmllint |
| uname | Linux 82ce33eaf23e 5.15.0-143-generic #153-Ubuntu SMP Fri Jun 13 19:10:45 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / c8c82fd703b1d589a7ab6fb5b922525ba0c9a66f |
| Default Java | Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/1/testReport/ |
| Max. process+thread count | 561 (vs. ulimit of 5500) |
| modules | C: hadoop-project U: hadoop-project |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/1/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 21s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  1s |  |  detect-secrets was not available.  |
| +0 :ok: |  xmllint  |   0m  1s |  |  xmllint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  24m 26s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 16s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  compile  |   0m 16s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  mvnsite  |   0m 18s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   0m 21s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   0m 16s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  |  46m 26s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 11s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 10s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javac  |   0m 10s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 10s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  javac  |   0m 10s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 13s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 11s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   0m 10s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| -1 :x: |  shadedclient  |   1m 52s |  |  patch has errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 10s |  |  hadoop-project in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 22s |  |  The patch does not generate ASF License warnings.  |
|  |   |  50m 54s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.51 ServerAPI=1.51 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/7785 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient codespell detsecrets xmllint |
| uname | Linux 8b2e81dcd2bc 5.15.0-143-generic #153-Ubuntu SMP Fri Jun 13 19:10:45 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / a33729c1b919f3c7d8f9c8dbe01d34a5f6b67fe5 |
| Default Java | Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/2/testReport/ |
| Max. process+thread count | 544 (vs. ulimit of 5500) |
| modules | C: hadoop-project U: hadoop-project |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/2/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

steveloughran commented:
aws failures (probably unrelated, given I'd not tested trunk yet)

```
[ERROR]   TestStagingPartitionedJobCommit.testReplaceWithDeleteFailure:259->verifyReplaceCommitActions:233 
Argument(s) are different! Wanted:
s3AFileSystem.delete(
    s3a://bucket-name/output/path/dateint=20161115/hour=14,
    true
);
-> at org.apache.hadoop.fs.s3a.S3AFileSystem.delete(S3AFileSystem.java:3538)
Actual invocations have different arguments:
s3AFileSystem.delete(
    s3a://bucket-name/output/path/dateint=20161116/hour=13,
    true
);
-> at org.apache.hadoop.fs.s3a.MockS3AFileSystem.delete(MockS3AFileSystem.java:272)
s3AFileSystem.delete(
    s3a://bucket-name/output/path/dateint=20161116/hour=14,
    true
);
-> at org.apache.hadoop.fs.s3a.MockS3AFileSystem.delete(MockS3AFileSystem.java:272)
s3AFileSystem.delete(
    s3a://bucket-name/output/path/dateint=20161115/hour=13,
    true
);
-> at org.apache.hadoop.fs.s3a.MockS3AFileSystem.delete(MockS3AFileSystem.java:272)
s3AFileSystem.delete(
    s3a://bucket-name/output/path/_temporary,
    true
);
-> at org.apache.hadoop.fs.s3a.MockS3AFileSystem.delete(MockS3AFileSystem.java:272)
```

steveloughran commented:
Surefire rejects me running without a named test
```
 mvit -Dparallel-tests -DtestsThreadCount=8 -Dtest=none
```

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.5.3:test (default-test) on project hadoop-aws: No tests matching pattern "none" were executed! (Set -Dsurefire.failIfNoSpecifiedTests=false to ignore this error.) -> [Help 1]
[ERROR] 
```

ayushtkn commented:
> Surefire rejects me running without a named test

that is a known issue some surefire upgrade maybe did that. We can put that in pom & get rid of this problem

https://github.com/apache/hadoop/pull/5567/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R126-R128

Looking at the previous build results, I think the Jenkins didn't run any test, we might need to touch something here & there to trigger all the tests

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 32s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  xmllint  |   0m  0s |  |  xmllint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  85m  2s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 21s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  compile  |   0m 21s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  mvnsite  |   0m 26s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   0m 26s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   0m 22s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  | 125m 52s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 14s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 12s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javac  |   0m 12s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 13s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  javac  |   0m 13s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 17s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 14s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   0m 13s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| -1 :x: |  shadedclient  |   3m 27s |  |  patch has errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 12s |  |  hadoop-project in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 30s |  |  The patch does not generate ASF License warnings.  |
|  |   | 132m 55s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.51 ServerAPI=1.51 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/7785 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient codespell detsecrets xmllint |
| uname | Linux 200bb3fe4862 5.15.0-139-generic #149-Ubuntu SMP Fri Apr 11 22:06:13 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / ac5300cfbf822a579168bec04f9029875b1289f2 |
| Default Java | Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/3/testReport/ |
| Max. process+thread count | 549 (vs. ulimit of 5500) |
| modules | C: hadoop-project U: hadoop-project |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/3/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

steveloughran committed:
HADOOP-19608. Upgrade to Junit 5.13.3

Build property trimStackTrace is set to false.

This is restores the behavior that any test failure
includes a deep stack trace rather than the first two elements.
In this version of surefire it also seems to print the full stack
of JUnit test execution and the java runtime itself.

This is only an issue for when tests fail.

Build property surefire.failIfNoSpecifiedTests is set to false.
This is required to retain the behavior where the invocation
    mvn verify -Dtest=unknown
will skip the unit tests but still run the integration tests, and a
specific property "it.test" can name the specific test to run:

  mvn verify -Dtest=unknown -Dit.test=ITestConnection

steveloughran commented:
tested s3 london. lots of Itests failing, which will need fixing after. That test `TestStagingPartitionedJobCommit` is good

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   8m 16s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  1s |  |  detect-secrets was not available.  |
| +0 :ok: |  xmllint  |   0m  1s |  |  xmllint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 1 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  13m  1s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  19m 44s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   8m 30s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  compile  |   7m 31s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  mvnsite  |   4m 27s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   3m 50s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   4m 10s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  |  82m 22s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 35s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m 32s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   8m  7s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javac  |   8m  7s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   7m 29s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  javac  |   7m 29s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   4m 19s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   3m 45s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   4m 12s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  |  34m 22s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 23s |  |  hadoop-project in the patch passed.  |
| +1 :green_heart: |  unit  |  18m 27s |  |  hadoop-common in the patch passed.  |
| -1 :x: |  unit  | 147m 51s | [/patch-unit-hadoop-hdfs-project.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/4/artifact/out/patch-unit-hadoop-hdfs-project.txt) |  hadoop-hdfs-project in the patch failed.  |
| +1 :green_heart: |  unit  |   2m 47s |  |  hadoop-aws in the patch passed.  |
| +1 :green_heart: |  unit  |   2m 35s |  |  hadoop-azure in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 31s |  |  hadoop-client-integration-tests in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 47s |  |  The patch does not generate ASF License warnings.  |
|  |   | 318m 32s |  |  |


| Reason | Tests |
|-------:|:------|
| Failed CTEST tests | test_test_libhdfs_ops_hdfs_static |
|   | test_test_libhdfs_threaded_hdfs_static |
|   | test_test_libhdfs_zerocopy_hdfs_static |
|   | test_test_native_mini_dfs |
|   | test_libhdfs_threaded_hdfspp_test_shim_static |
|   | test_hdfspp_mini_dfs_smoke_hdfspp_test_shim_static |
|   | libhdfs_mini_stress_valgrind_hdfspp_test_static |
|   | memcheck_libhdfs_mini_stress_valgrind_hdfspp_test_static |
|   | test_libhdfs_mini_stress_hdfspp_test_shim_static |
|   | test_hdfs_ext_hdfspp_test_shim_static |
| Failed junit tests | hadoop.hdfs.server.federation.security.token.TestZKDelegationTokenSecretManagerImpl |
|   | hadoop.hdfs.server.federation.router.TestRouterMountTableCacheRefresh |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.51 ServerAPI=1.51 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/7785 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient codespell detsecrets xmllint |
| uname | Linux 1edd78deb703 5.15.0-143-generic #153-Ubuntu SMP Fri Jun 13 19:10:45 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 92484abbddff32ce1d550493e19b378ac8687703 |
| Default Java | Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
| CTEST | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/4/artifact/out/patch-hadoop-hdfs-project-ctest.txt |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/4/testReport/ |
| Max. process+thread count | 4920 (vs. ulimit of 5500) |
| modules | C: hadoop-project hadoop-common-project/hadoop-common hadoop-hdfs-project hadoop-tools/hadoop-aws hadoop-tools/hadoop-azure hadoop-client-modules/hadoop-client-integration-tests U: . |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/4/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

steveloughran commented:
HDFS tests failing. Of the two classes run in my IDE

* TestRouterMountTableCacheRefresh.testMountTableEntriesCacheUpdateTimeout times out for option 2, but not "1"
* TestZKDelegationTokenSecretManagerImpl passes

Assuming unrelated and cutting the hdfs test run trigger

pan3793 commented on hadoop-project/pom.xml:
```diff
@@ -1315,11 +1324,6 @@
-      <dependency>
-        <groupId>org.mockito</groupId>
-        <artifactId>mockito-junit-jupiter</artifactId>
-        <version>3.12.4</version>
```
is this change necessary? or just clean up unused dependency declarations

steveloughran replied:
I was getting a warning of a double reference, and while I was doing jupiter stuff, thought I should do this too

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 19s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  xmllint  |   0m  0s |  |  xmllint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 1 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |   9m 35s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  19m 33s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   8m 33s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  compile  |   7m 29s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  mvnsite  |   4m 34s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   3m 50s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   4m  8s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  |  78m 38s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 29s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m 33s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   8m 10s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javac  |   8m 10s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   7m 27s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  javac  |   7m 27s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   4m 22s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   3m 47s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   4m  5s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  |  34m 30s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 24s |  |  hadoop-project in the patch passed.  |
| +1 :green_heart: |  unit  |  18m 29s |  |  hadoop-common in the patch passed.  |
| -1 :x: |  unit  | 168m 12s | [/patch-unit-hadoop-hdfs-project.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/6/artifact/out/patch-unit-hadoop-hdfs-project.txt) |  hadoop-hdfs-project in the patch failed.  |
| +1 :green_heart: |  unit  |   3m  1s |  |  hadoop-aws in the patch passed.  |
| +1 :green_heart: |  unit  |   2m 31s |  |  hadoop-azure in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 34s |  |  hadoop-client-integration-tests in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 47s |  |  The patch does not generate ASF License warnings.  |
|  |   | 327m 23s |  |  |


| Reason | Tests |
|-------:|:------|
| Failed CTEST tests | test_test_libhdfs_ops_hdfs_static |
|   | test_test_libhdfs_threaded_hdfs_static |
|   | test_test_libhdfs_zerocopy_hdfs_static |
|   | test_test_native_mini_dfs |
|   | test_libhdfs_threaded_hdfspp_test_shim_static |
|   | test_hdfspp_mini_dfs_smoke_hdfspp_test_shim_static |
|   | libhdfs_mini_stress_valgrind_hdfspp_test_static |
|   | memcheck_libhdfs_mini_stress_valgrind_hdfspp_test_static |
|   | test_libhdfs_mini_stress_hdfspp_test_shim_static |
|   | test_hdfs_ext_hdfspp_test_shim_static |
| Failed junit tests | hadoop.hdfs.server.federation.security.token.TestZKDelegationTokenSecretManagerImpl |
|   | hadoop.hdfs.server.federation.router.TestRouterMountTableCacheRefresh |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.51 ServerAPI=1.51 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/6/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/7785 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient codespell detsecrets xmllint |
| uname | Linux bc9477951bf4 5.15.0-143-generic #153-Ubuntu SMP Fri Jun 13 19:10:45 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / c1d727a3d83a67013a46ee56b445ce0f8e9bb6c9 |
| Default Java | Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
| CTEST | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/6/artifact/out/patch-hadoop-hdfs-project-ctest.txt |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/6/testReport/ |
| Max. process+thread count | 3977 (vs. ulimit of 5500) |
| modules | C: hadoop-project hadoop-common-project/hadoop-common hadoop-hdfs-project hadoop-tools/hadoop-aws hadoop-tools/hadoop-azure hadoop-client-modules/hadoop-client-integration-tests U: . |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/6/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

steveloughran commented:
Last forced commit squashed all work and cut out the changes to trigger the hdfs and commons builds -just updates the project pom and adds dsocs.

can I get this in ASAP, as it is needed for #7814 and any other PR trying to run fs contract tests parameterized.

ayushtkn reviewed (approved):
If the build doesn't complains. Changes LGTM

ayushtkn commented on hadoop-common-project/hadoop-common/src/test/resources/test-fake-default.xml:
```diff
@@ -26,4 +26,5 @@
     <value>tests.fake-default.value</value>
     <description>a default value for the "new" key of a deprecated pair.</description>
   </property>
+
```
i think this is some leftover? if so, do drop it before you commit

steveloughran replied:
yeah, I missed that, and yes, its to be dropped. It'll force hadoop-common tests to run as the final safety check

hadoop-yetus commented:
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 21s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  xmllint  |   0m  0s |  |  xmllint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 1 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |   8m 47s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  19m 19s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   8m 34s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  compile  |   7m 23s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  mvnsite  |   2m 59s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   2m 39s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   2m 33s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  |  73m 16s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 28s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   1m 38s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   8m  9s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javac  |   8m  9s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   7m 31s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  javac  |   7m 31s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   2m 19s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   1m 55s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   2m 34s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  |  28m 55s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 26s |  |  hadoop-project in the patch passed.  |
| +1 :green_heart: |  unit  |  18m 36s |  |  hadoop-common in the patch passed.  |
| +1 :green_heart: |  unit  |   2m 44s |  |  hadoop-aws in the patch passed.  |
| +1 :green_heart: |  unit  |   2m 29s |  |  hadoop-azure in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 27s |  |  hadoop-client-integration-tests in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 41s |  |  The patch does not generate ASF License warnings.  |
|  |   | 147m 11s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.51 ServerAPI=1.51 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/7/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/7785 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient codespell detsecrets xmllint |
| uname | Linux 0012f0a0adcb 5.15.0-143-generic #153-Ubuntu SMP Fri Jun 13 19:10:45 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / ffbf3b00f97e764e888fc1b0a66738f17d3dd3bf |
| Default Java | Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/7/testReport/ |
| Max. process+thread count | 1326 (vs. ulimit of 5500) |
| modules | C: hadoop-project hadoop-common-project/hadoop-common hadoop-tools/hadoop-aws hadoop-tools/hadoop-azure hadoop-client-modules/hadoop-client-integration-tests U: . |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/7/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

slfan1989 commented:
@steveloughran LGTM. After #7814 is completed, I will proceed with fixing the parameterized test issues in other modules.

steveloughran commented:
@slfan1989 thanks, should be good soon, though miniyarn clusters aren't playing (think it's unrelated).

And I know a lot more about JUnit 5, even though I think the team don't value developer's existing codebase *or knowledge* enough. Most unhappy about
- changed ordering of assertEquals with message.
- the way that @Test, @BeforeEach, @AfterEach aren't inherited any more. Junit 1 only had `setUp()` as the setup method, and so those of us who started coding that way have expectations no longer being met.


At least our adoption of AssertJ was prescient.

steveloughran committed:
HADOOP-19608. Remove test trigger change

hadoop-yetus commented:
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 33s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  xmllint  |   0m  0s |  |  xmllint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 1 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |   8m 19s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  34m 35s |  |  trunk passed  |
| +1 :green_heart: |  compile  |  17m 53s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  compile  |  15m 16s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  mvnsite  |   4m 46s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   4m 19s |  |  trunk passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   3m 36s |  |  trunk passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  | 127m 55s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 41s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m 37s |  |  the patch passed  |
| +1 :green_heart: |  compile  |  16m  6s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javac  |  16m  6s |  |  the patch passed  |
| +1 :green_heart: |  compile  |  14m 15s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  javac  |  14m 15s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   4m 55s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   4m  8s |  |  the patch passed with JDK Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   3m 34s |  |  the patch passed with JDK Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09  |
| +1 :green_heart: |  shadedclient  |  54m 49s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 36s |  |  hadoop-project in the patch passed.  |
| +1 :green_heart: |  unit  |  23m 23s |  |  hadoop-common in the patch passed.  |
| +1 :green_heart: |  unit  |   3m 45s |  |  hadoop-aws in the patch passed.  |
| +1 :green_heart: |  unit  |   3m  9s |  |  hadoop-azure in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 36s |  |  hadoop-client-integration-tests in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m  6s |  |  The patch does not generate ASF License warnings.  |
|  |   | 251m 43s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.51 ServerAPI=1.51 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/8/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/7785 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient codespell detsecrets xmllint |
| uname | Linux bda969a551f6 5.15.0-139-generic #149-Ubuntu SMP Fri Apr 11 22:06:13 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / c470c01900d0b881a9a1c0e2e9fe3ba2294614c5 |
| Default Java | Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.27+6-post-Ubuntu-0ubuntu120.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_452-8u452-ga~us1-0ubuntu1~20.04-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/8/testReport/ |
| Max. process+thread count | 1272 (vs. ulimit of 5500) |
| modules | C: hadoop-project hadoop-common-project/hadoop-common hadoop-tools/hadoop-aws hadoop-tools/hadoop-azure hadoop-client-modules/hadoop-client-integration-tests U: . |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7785/8/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.
